AmplifyPlugins/DataStore/Sources/AWSDataStorePlugin/Storage/CascadeDeleteOperation.swift

```diff
@@ -137,12 +157,13 @@ public class CascadeDeleteOperation<M: Model>: AsynchronousOperation {
             }
             
             return collapseResults(queryResult: queriedResult,
-                                                    deleteResult: deletedResult,
-                                                    associatedModels: associatedModels)
+                                   deleteResult: deletedResult,
+                                   associatedModels: associatedModels)
         }
         
         let modelIds = queriedModels.map { $0.identifier(schema: self.modelSchema).stringValue }
-
+        _log("[CascadeDelete.1] Deleting \(modelSchema.name) with identifiers: \(modelIds)")
```

Comment:
is the number inside `[CascadeDelete.1]` define the order should be followed?

Reply:
Yes, looks like it.  Though there are repeated numbers, but they represent the same task and follows the sequence.